clearwater-config-manager.root/usr/share/clearwater/clearwater-config-manager/scripts/upload_shared_config

```diff
@@ -37,6 +37,21 @@ local_site_name=site1
 etcd_key=clearwater
 . /etc/clearwater/config
 
+[ $# -le 1 ] || { echo "Usage: upload_shared_config [--force]" >&2 ; exit 2 ; }
```

Comment:
Just to check i'm reading this right, does this statement simply print the usage and then exit if the number of arguments provided is not less than or equal to 1?
That is, if the arg number is 0 or 1, the first part of the OR evaluates true and so we drop out before touching the other branch?